src/styles/_catalog.scss

```
@@ -1,22 +1,40 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 48px;
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
 }
+
+@media (max-width: 1023.9px) {
```

you don't have to add decimals. just leave 1023px
```suggestion
@media (max-width: 1023px) {
```

---

please check all places